packages/devextreme/js/__internal/core/state_manager/state_manager_factory.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable spellcheck/spell-checker */
+import { Logger } from './logger';
+import { ObservableValueContainerManager } from './observable_value_container_manager';
+import { ReduxDevToolsConnector } from './redux_dev_tools_connector';
+import { StateManager } from './state_manager';
+import type * as StateManagementTypes from './types';
+
+export const makeStateManager = (
+  options: StateManagementTypes.StateManagerFactoryOptions,
+): StateManager => {
+  const logger = options.logger ?? new Logger({ logLevel: options.logLevel, prefix: '[StateManager]' });
+
+  const stateContainerManagers: StateManagementTypes.StateManagerConfig['valueContainerManagers'] = options.valueContainerManagers ?? [ObservableValueContainerManager];
```

Comment:
Consider using the type directly
```suggestion
  const stateContainerManagers: ValueContainerManagerConstructor[] = options.valueContainerManagers ?? [ObservableValueContainerManager];
```

Reply:
changed